### DISCUSSION TITLE
常青笔记

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#69] [主发布站的评论系统]
# 常青笔记

%%我的理解:: 常青笔记就是andy基于[[🧑卢曼]]的卡片盒笔记法，结合现代计算机软件的优势和自己的理解和实践，衍生出的综合记笔记理念：要么不记，要记就记常青的笔记。%%
写下常青笔记并加以组织，意在时过境迁，世事辗转之中，这些笔记仍能演化改良，发挥功用，最终聚沙成塔。如此考虑笔记实属不寻常：[[大多数人只写临时的笔记]]。写常青笔记，与其说是写笔记，不如说是高效发展洞察力：[[更好地记笔记这个描述太过流于表面；核心在于更好地思考]]。常青笔记写得好则会颇富价值：[[常青笔记是知识工作的基本单位]]。
写出经得住长久研磨改善的笔记，实属不易。下面这些原则可能有所帮助：
 [[常青笔记应该是原子化的]] [[常青笔记应面向概念]] [[常青笔记之间应该建立大量链接]] [[优先建立信息间的联系来分类，而非使用层级式分类法]] [[默认面对自己写笔记，不要考虑有读者]]  常青笔记这一概念在很大程度上是由尼克拉斯-卢曼的 卡片盒笔记法演变而来，尼克拉斯-卢曼认为，在他 70 本书的写作过程中，Zettelkasten 就像一个独立的智力伙伴。参见[[常青笔记和卡片盒笔记法的异同点]]
实现常青笔记 参见：
 [[写你所读，以求内化]] [[为有潜在价值的参考引用建立一个阅读收件箱]] [[为临时性的和不完整的笔记设置写作收件箱]] [[可执行的写作策略]]   参考文献 Ahrens, S.

https://garden.oldwinter.top/%E5%B8%B8%E9%9D%92%E7%AC%94%E8%AE%B0

**Comment by CatsOfMrXue** [comment]
在知乎看到您关于常青笔记的文章，但是知乎私信关闭了，很高兴遇到您这样愿意分享自己的独特思考的博主，十分受用
